README.md

```diff
@@ -52,7 +52,7 @@ config.omniauth :openid_connect, {
 | name                         | Arbitrary string to identify connection and identify it from other openid_connect providers                                                                                                                        | no       | String: openid_connect     | :my_idp                                             |
 | issuer                       | Root url for the authorization server                                                                                                                         | yes      |                            | https://myprovider.com                              |
 | discovery                    | Should OpenID discovery be used. This is recommended if the IDP provides a discovery endpoint. See client config for how to manually enter discovered values. | no       | false                      | one of: true, false                                 |
-| client_auth_method           | Which authentication method to use to authenticate your app with the authorization server                                                                     | no       | Sym: basic                 | "basic", "jwks"                                     |
+| client_auth_method           | Which authentication method to use to authenticate your app with the authorization server                                                                     | no       | Sym: basic                 | "basic", "jwt_bearer"                               |
```

Comment:
According to https://github.com/nov/rack-oauth2/blob/1ba7dc7ba4360dedfa12643bf12dbaae4e6293c1/lib/rack/oauth2/client.rb#L140-L186, the valid options:

* `basic`
* `basic_without_www_form_urlencode`
* `jwt_bearer`
* `saml2_bearer`
* `mtls`
* anything else is ignored